emulators/mame/Portfile

```
@@ -66,20 +176,328 @@ build.args-append   ARCHOPTS="${configure.cxxflags}" \
+    copy ${worksrcpath}/hlsl ${mame_target_dir}
+    copy ${worksrcpath}/ini ${mame_target_dir}
+    xinstall -m 755 -d ${mame_target_dir}/inp
+    copy ${worksrcpath}/keymaps ${mame_target_dir}
+    copy ${worksrcpath}/language ${mame_target_dir}
+    xinstall -m 755 -d ${mame_target_dir}/nvram
+    copy ${worksrcpath}/plugins ${mame_target_dir}
+    copy ${worksrcpath}/roms ${mame_target_dir}
+    copy ${worksrcpath}/samples ${mame_target_dir}
+    copy ${worksrcpath}/scripts ${mame_target_dir}
+    xinstall -m 755 -d ${mame_target_dir}/software
+    xinstall -m 755 -d ${mame_target_dir}/snap
+    xinstall -m 755 -d ${mame_target_dir}/sta
+
+    # Docs
+    xinstall -m 755 -d ${mame_share_dir}
+    copy ${worksrcpath}/docs/build/man ${mame_share_dir}
+
+    # Tools
+    if {[variant_isset tools]} {
+        xinstall -m 755 \
+            ${worksrcpath}/aueffectutil \
+            ${worksrcpath}/castool \
+            ${worksrcpath}/chdman \
+            ${worksrcpath}/floptool \
+            ${worksrcpath}/imgtool \
+            ${worksrcpath}/jedutil \
+            ${worksrcpath}/ldresample \
+            ${worksrcpath}/ldverify \
+            ${worksrcpath}/nltool \
+            ${worksrcpath}/nlwav \
+            ${worksrcpath}/pngcmp \
+            ${worksrcpath}/regrep \
+            ${worksrcpath}/romcmp \
+            ${worksrcpath}/split \
+            ${worksrcpath}/srcclean \
+            ${worksrcpath}/testkeys \
+            ${worksrcpath}/unidasm \
+            ${mame_target_dir}
+    }
+
+    unset mame_share_dir
+    unset mame_target_dir
+}
+
+post-destroot {
+    set mp_man_dest_dir "${destroot}${prefix}/share/man/man1"
+    set mame_target_dir "${destroot}${prefix}/libexec/mame"
+    set mame_share_dir "${destroot}${prefix}/share/mame"
+    set mame_launch_script "${mame_target_dir}/mame.sh"
+    set mame_launch_link "${destroot}${prefix}/bin/mame"
+
+    # Create launch wrapper script
+    xinstall -m 755 "${distpath}/mame-script-template-launcher.sh" \
+        ${mame_launch_script}
+    # Substitute placeholder with real executable name
+    reinplace "s|@@MACPORTS_MAME_EXECUTABLE@@|${executable}|g" \
+        ${mame_launch_script}
+
+    # Create soft link 'mame', to launch wrapper script
+    ln -s "${prefix}/libexec/mame/mame.sh" \
+        ${mame_launch_link}
+
+    # Create soft link to man page
+    xinstall -d -m 755 ${mp_man_dest_dir}
+    ln -s "${prefix}/share/mame/man/MAME.1" \
+        "${mp_man_dest_dir}/mame.1"
+
+    unset mame_launch_link
+    unset mame_launch_script
+    unset mame_share_dir
+    unset mame_target_dir
+    unset mp_man_dest_dir
+}
+
+#------------------------------------------------------------------------------
+# Notes
+#------------------------------------------------------------------------------
+
+notes {
+    Mame is launched via command 'mame'.
+
+    If a blank screen is encountered, press ESC to exit, and then re-launch
+    with an alternative video option.
```

Don't include literal newlines in the middle of paragraphs. Let MacPorts word wrap to the terminal width.
```suggestion
    If a blank screen is encountered, press ESC to exit, and then re-launch\
    with an alternative video option.
```